sign/README.md

```diff
@@ -0,0 +1,13 @@
+# sign 
+`sign` package represents the API and signals for sending and receiving
+signature request to and from our API user.
+
+When a method is called that requires an additional signature confirmation from
+a user (like, a transaction), it gets it's sign request.
+
+Client of the API is then nofified of the sign request.
+
+Client has a chance to approve the sign request (by providing a valid password)
+or to discard it. When the request is approved, the locked functinality is
+executed.
+
```

Comment:
Shouldn't README packages should be  generated with autoreadme like on [geth/jail package](https://github.com/status-im/status-go/blob/develop/geth/jail/doc.go)

Reply:
good point

Reply:
@adriacidre hmm, it looks like we don't have a makefile target or something to do that.
let's skip it for this PR then.